src/Meilisearch/Constants.cs

```diff
@@ -0,0 +1,29 @@
+namespace Meilisearch
+{
+    using System.Text.Json;
+    using System.Text.Json.Serialization;
+
+    /// <summary>
+    /// This class adds some defaults to work with Meilisearch client.
+    /// </summary>
+    public static class Constants
+    {
+        /// <summary>
+        /// JsonSerializer options used when serializing objects that needs to remove null values.
+        /// </summary>
+        public static readonly JsonSerializerOptions JsonSerializerOptionsRemoveNulls = new JsonSerializerOptions
+        {
+            DefaultIgnoreCondition = JsonIgnoreCondition.WhenWritingNull,
+            PropertyNamingPolicy = JsonNamingPolicy.CamelCase,
+        };
+
+        /// <summary>
+        /// JsonSerializer options used when serializing objects that keeps null values.
+        /// </summary>
+        public static readonly JsonSerializerOptions JsonSerializerOptionsWriteNulls = new JsonSerializerOptions
```

Comment:
What do you think to use this one inside of the `CreateKeyAsync` method?

src/Meilisearch/MeilisearchClient.cs:315